front/app/Components/AREAForm.tsx

```diff
@@ -0,0 +1,64 @@
+/**
+ * @client
+ */
+import React, {useState} from 'react';
+
+/* Interfaces */
+import {ActionJsonArray} from "@/app/Interfaces/ActionJson.interface";
+
+/**
+ * @component AREAForm component
+ * @param {ActionJsonArray} fields - Fields to display
+ * @param setDatas
+ * @return {JSX.Element} AREAForm component with fields to display
+ * @note This component is used to display a form with fields and submit button that will return the data in JSON format
+ */
+const AREAForm: React.FC<{ fields: ActionJsonArray, setDatas: (data: string) => void }> = ({fields, setDatas}) => {
+    const [formData, setFormData] = useState<{ [key: string]: string }>({});
+    const [error, setError] = useState<string | null>(null);
```

Comment:
Should be no spaces around curly brackets.
```suggestion
const AREAForm: React.FC<{fields: ActionJsonArray, setDatas: (data: string) => void}> = ({fields, setDatas}) => {
    const [formData, setFormData] = useState<{[key: string]: string}>({});
    const [error, setError] = useState<string | null>(null);
```